Userland/Utilities/abench.cpp

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (c) 2021, the SerenityOS developers.
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <AK/NumericLimits.h>
+#include <AK/Types.h>
+#include <LibAudio/Loader.h>
+#include <LibCore/ArgsParser.h>
+#include <LibCore/ElapsedTimer.h>
+#include <LibCore/File.h>
+#include <LibCore/System.h>
+#include <LibMain/Main.h>
+#include <stdio.h>
+
+// The Kernel has problems with large anonymous buffers, so let's limit sample reads ourselves.
+static constexpr size_t MAX_CHUNK_SIZE = MiB / 2;
```

Comment:
`MiB / 2` is a funny thing to say, please at least type out `1 * MiB / 2` to make it make sense.